src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const section = document.querySelector('.chess');
+
+document.addEventListener('DOMContentLoaded', () => {
+  const div = document.createElement('div');
+
+  section.append(div);
+  div.className = 'chess__field';
+
+  const colors = ['chess__white', 'chess__black'];
+
+  for (let i = 0; i < 8; i++) {
+    for (let j = 0; j < 8; j++) {
+      const cells = document.createElement('div');
+
+      if (i % 2 === j % 2) {
```

Comment:
Can you create a separate function for this functionality? Something like `isBlackCell`